cmd/crank/beta/xpkg/init.go

```diff
@@ -228,6 +228,7 @@ func printFile(w io.Writer, path string) error {
 	if err != nil {
 		return errors.Wrapf(err, "failed to open file %s", path)
 	}
+	defer f.Close()
```

Comment:
errcheck linter complains about unchecked errors, I think here it's safe to just ignore
```suggestion
	//nolint:errcheck // We don't care about the error returned here.
	defer f.Close()
```